scripts/apps/publish/views/subscribers.html

```diff
@@ -265,12 +268,9 @@ <h3 class="modal__heading" translate ng-hide="subscriber._id">Add New Subscriber
                                         <div class="sd-line-input sd-line-input--is-select">
                                             <select class="sd-line-input__select"
                                                 ng-model="tokens.ttl"
+                                                ng-disabled="tokens.neverExpire"
                                                 title="{{ :: 'Token expiry' | translate }}">
-                                                <option value="7" translate>1 week</option>
-                                                <option value="14" translate>2 weeks</option>
-                                                <option value="30" translate>1 month</option>
-                                                <option value="180" translate>6 months</option>
-                                                <option value="365" translate>1 year</option>
+                                                <option ng-repeat="(key, value) in tokens.days" value="{{value}}" translate>{{key}}</option>
```

Comment:
now that values are dynamic we lose the `translate` feature, don't we?

Reply:
Yes, you are right.